cura/PrinterOutput/Models/PrinterOutputModel.py

```diff
@@ -290,7 +293,7 @@ def canUpdateFirmware(self) -> bool:
     def _onControllerCanUpdateFirmwareChanged(self) -> None:
         self.canUpdateFirmwareChanged.emit()
 
-    # Returns the configuration (material, variant and buildplate) of the current printer
+    # Returns the active configuration (material, variant and buildplate) of the current printer
     @pyqtProperty(QObject, notify = configurationChanged)
     def printerConfiguration(self) -> Optional[PrinterConfigurationModel]:
         if self._printer_configuration.isValid():
```

Comment:
Changing this property name will break the API, but I think we can change `self._printer_configuration` to `self._active_printer_configuration` so internally the naming will be more clear.

Reply:
Fair enough.